### ISSUE TITLE
vo_checkbox counts deferred items as done

### ISSUE THREAD

**Issue by mcepl** [vimoutliner/vimoutliner#61]
This is a part of outline with TODO list for getting all my papers back when I lost my wallet (you don't need to understand the text, and all phone numbers etc. are of public institutions):

```
-[_] Náhrada peněženky                                                       
-    [_] 50% občanka                                                             
2        [X] nahlásit ztrátu, tel 222 116 111 ?                                  
2        [_] nová bude ???                                                       
|    [X] fotografie                                                              
-    [X] 100% řidičák                                                            
2        [X] v Jungmance, pas, foto a potvrzení                                  
2        [-] vyzvednout 30.9. v Business Centrum Vyšehrad, Na Pankráci 17        
-    [X] malý techničák                                                          
2        [X] na Jarov, pas, potvrzení, velký techničák                           
-    [X] 100% InKarta ČD                                                         
2        [X] dojít na nádraží, zablokovat, objednat                              
2        [-] dojde kolem 10.10. měla by být začátkem října
```

I have a problem with `[-]` items. "InKarta ČD" is still waiting on the Czech railways actually delivering me the card back (that's the last item), so the total of this item should not be 100%, because it is not done yet.
